sklearn/cluster/_affinity_propagation.py

```diff
@@ -258,8 +258,8 @@ def affinity_propagation(
 
     Notes
     -----
-    For an example, see :ref:`examples/cluster/plot_affinity_propagation.py
-    <sphx_glr_auto_examples_cluster_plot_affinity_propagation.py>`.
+    For an example usage,
+    see :ref:`sphx_glr_auto_examples_cluster_plot_affinity_propagation.py`.
```

Comment:
I would also mention this example here: https://scikit-learn.org/stable/auto_examples/applications/plot_stock_market.html#clustering-using-affinity-propagation

Reply:
I have added the example, and rebased the branch.